src/target/common/devo/protocol/uart_ram.c

```diff
@@ -0,0 +1,43 @@
+/*
+    This project is free software: you can redistribute it and/or modify
```

Comment:
where this file get referenced? when we compile mod, we don't auto include this file.

Reply:
Files in this directory are included in the .mod builds for modular builds, and also in a normal build.  See lines 24-32 of target/common/devo/Makefile.inc.

Reply:
okay. I will change my PR #464